python-usfm-parser/README.md

```diff
@@ -11,6 +11,8 @@ Built on python 3.10
 
 `pip install usfm-grammar`
 
+This requires a compiler. On Windows, Microsoft Visual C++ 14.0 or above is required. 
```

Comment:
IIUC this would only be needed for actually building the package from source and not for simple installation via `pip` now that we providing pre-built wheels. Is that correct?

Reply:
No, we would still need a C compiler. [This requirement](https://github.com/tree-sitter/py-tree-sitter#installation) comes from the `tree-sitter` package which is our dependency. On systems where we have a C compiler persent(like gcc) this would install without any fuss.

The dependecies are not packed inside our wheel right?

Reply:
I think we'll need to test this further. I think we can assume this for now.